src/Turn.js

```diff
@@ -1,29 +1,27 @@
 import domUpdates from './domUpdates';
 
 class Turn {
-  constructor(game, round) {
-    this.game = game;
-    this.round = round;
-    this.currentPlayer = this.game.playerOne;
+  constructor() {
   }
 
-  checkGuess(guess) {
-    let index = this.round.currentSurveyAnswers.findIndex(answerObj => answerObj.answer.toLowerCase() === guess.toLowerCase());
+  checkGuess(game, guess) {
+    let index = game.currentRound.currentSurveyAnswers.findIndex(answerObj => answerObj.answer.toLowerCase() === guess.toLowerCase());
     if (index !== -1) {
-      let targetAnswer = this.round.currentSurveyAnswers.splice(index, 1)[0];
-      this.assignPoints(targetAnswer);
+      let targetAnswer = game.currentRound.currentSurveyAnswers.splice(index, 1)[0];
+      this.assignPoints(game, targetAnswer);
     }
-    this.endTurn();
+    this.endTurn(game);
   }
 
-  assignPoints(targetAnswer) {
-    this.currentPlayer.score += targetAnswer.respondents * this.currentPlayer.multiplier;
+  assignPoints(game, targetAnswer) {
+    game.currentRound.currentPlayer.score += targetAnswer.respondents * game.currentRound.currentPlayer.multiplier;
```

Comment:
You’re digging pretty deep into objects here, this is a good use case for destructuring. e.g:

```js
let { score, multiplier } = game.currentRound.currentPlayer;
score += targetAnswer.respondents * multiplier;
```

Reply:
We really like that idea, but we couldn't get it to work. We'd love to check in with you about it, though.

Reply:
Ooooh actually it might not, I think the game instance’s score wouldn’t actually get updated because now we’re creating a brand new variable `score` in memory. So although that variable is updated, it’s not transferring back to the game instance. MY BAD sorry for leading you astray!